modules/imgcodecs/src/grfmt_tiff.cpp

```diff
@@ -1196,7 +1196,7 @@ static bool readParam(const std::vector<int>& params, int key, int& value)
     return false;
 }
 
-bool TiffEncoder::writeLibTiff( const std::vector<Mat>& img_vec, const std::vector<int>& params)
```

Comment:
I propose not touch tiff for now as it's not animation, there are no timings and technically the pages may have different resolution.